extensions/integration-tests/src/main.ts

```diff
@@ -34,26 +35,71 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 
 	vscode.commands.registerCommand('test.waitForExtensionsToLoad', async () => {
-		let expectedExtensions = ['Microsoft.agent', 'Microsoft.import', 'Microsoft.mssql', 'Microsoft.profiler'];
+		const expectedExtensions = ['Microsoft.agent', 'Microsoft.import', 'Microsoft.mssql', 'Microsoft.profiler', 'Microsoft.azuredatastudio-postgresql'];
+		const commonFeatures: azdata.DataProviderType[] = [
+			azdata.DataProviderType.AdminServicesProvider,
+			azdata.DataProviderType.BackupProvider,
+			azdata.DataProviderType.CapabilitiesProvider,
+			azdata.DataProviderType.ConnectionProvider,
+			azdata.DataProviderType.FileBrowserProvider,
+			azdata.DataProviderType.MetadataProvider,
+			azdata.DataProviderType.ObjectExplorerProvider,
+			azdata.DataProviderType.ProfilerProvider,
+			azdata.DataProviderType.QueryProvider,
+			azdata.DataProviderType.RestoreProvider,
+			azdata.DataProviderType.ScriptingProvider,
+			azdata.DataProviderType.TaskServicesProvider];
+
+		const features_mssql: azdata.DataProviderType[] = [
+			azdata.DataProviderType.AgentServicesProvider,
+			azdata.DataProviderType.DacFxServicesProvider,
+			azdata.DataProviderType.IconProvider,
+			azdata.DataProviderType.SchemaCompareServicesProvider
+		];
+
+		features_mssql.push(...commonFeatures);
+
+		const providerFeatureMapping: { providerId: string, features: azdata.DataProviderType[] }[] = [
+			{
+				providerId: 'MSSQL',
```

Comment:
Same for this - we should find a way to track the registered provider IDs to ensure that when new ones are added the test ensures they're also added here